website/docs/plugin/framework/migrating/attributes-blocks/fields.mdx

```diff
@@ -7,37 +7,37 @@ description: >-
+| ForceNew              | [RequiresReplace](/terraform/plugin/framework/migrating/attributes-blocks/force-new) on `PlanModifiers` field on attribute or implementation of [ResourceWithModifyPlan](/terraform/plugin/framework/migrating/resources/plan-modification#framework) interface                                                                                                                                                                                                                            |
+| DiffSuppressFunc      | [PlanModifiers](https://terraform.io/framework/migrating/resources/plan-modification#framework) field on attribute or implementation of [ResourceWithModifyPlan](/terraform/plugin/framework/migrating/resources/plan-modification#framework) interface                                                                                                                                                                                                                                              |
+| DiffSuppressOnRefresh | [Read](/terraform/plugin/framework/migrating/resources/crud) method on resource                                                                                                                                                                                                                                                                                                                                                                                                  |
+| Default               | [PlanModifiers](https://terraform.io/framework/migrating/resources/plan-modification#framework) field on attribute or implementation of [ResourceWithModifyPlan](/terraform/plugin/framework/migrating/resources/plan-modification#framework) interface                                                                                                                                                                                                                                              |
+| DefaultFunc           | [PlanModifiers](https://terraform.io/framework/migrating/resources/plan-modification#framework) field on attribute or implementation of [ResourceWithModifyPlan](/terraform/plugin/framework/migrating/resources/plan-modification#framework) interface                                                                                                                                                                                                                                              |
```

Comment:
The migration here was doing the right thing, but the old links were missing the /plugin/ prefix -- so adjusting them to /terraform/plugin/ instead of terraform.io

```suggestion
| DiffSuppressFunc      | [PlanModifiers](/terraform/plugin/framework/migrating/resources/plan-modification#framework) field on attribute or implementation of [ResourceWithModifyPlan](/terraform/plugin/framework/migrating/resources/plan-modification#framework) interface                                                                                                                                                                                                                                              |
| DiffSuppressOnRefresh | [Read](/terraform/plugin/framework/migrating/resources/crud) method on resource                                                                                                                                                                                                                                                                                                                                                                                                  |
| Default               | [PlanModifiers](/terraform/plugin/framework/migrating/resources/plan-modification#framework) field on attribute or implementation of [ResourceWithModifyPlan](/terraform/plugin/framework/migrating/resources/plan-modification#framework) interface                                                                                                                                                                                                                                              |
| DefaultFunc           | [PlanModifiers](/terraform/plugin/framework/migrating/resources/plan-modification#framework) field on attribute or implementation of [ResourceWithModifyPlan](/terraform/plugin/framework/migrating/resources/plan-modification#framework) interface                                                                                                                                                                                                                                              |
```